Assets/Tests/InputSystem/CorePerformanceTests.cs

```diff
@@ -1113,7 +1113,9 @@ public void Performance_OptimizedControls_ReadingPose4kTimes(OptimizationTestTyp
         "InputSystem.onAfterUpdate",
         "PreUpdate.NewInputUpdate",
         "PreUpdate.InputForUIUpdate",
-        "FixedUpdate.NewInputFixedUpdate"
+        "FixedUpdate.NewInputFixedUpdate",
+        "InputAction.Enable",
+        "InputActionMap.ResolveBindings"
```

Comment:
please also add "InputAction.Disable" here, thanks

Reply:
added in 539258b